pastastore/base.py

```diff
@@ -911,6 +1009,48 @@ def get_models(
         else:
             return models
 
+    def get_model(
+        self,
+        names: Union[list, str],
+        return_dict: bool = False,
+        progressbar: bool = False,
+        squeeze: bool = True,
+        update_ts_settings: bool = False,
+    ) -> Union[ps.Model, list]:
+        """Load models from database.
+
+        Alias for get_models().
+
+        Parameters
+        ----------
+        names : str or list of str
+            names of the models to load
+        return_dict : bool, optional
+            return model dictionary instead of pastas.Model (much
+            faster for obtaining parameters, for example)
+        progressbar : bool, optional
+            show progressbar, by default False
+        squeeze : bool, optional
+            if True return Model instead of list of Models
+            for single entry
+        update_ts_settings : bool, optional
+            update time series settings based on time series in store.
+            overwrites stored tmin/tmax in model.
+
+        Returns
+        -------
+        pastas.Model or list of pastas.Model
+            return pastas model, or list of models if multiple names were
+            passed
+        """
+        return self.get_models(
+            names,
+            return_dict=return_dict,
+            progressbar=progressbar,
+            squeeze=squeeze,
+            update_ts_settings=update_ts_settings,
+        )
+
```

Comment:
Maybe we can copy the docstring such that we only have to maintain one function? 

E.g. https://stackoverflow.com/questions/68901049/copying-the-docstring-of-function-onto-another-function-by-name

Reply:
Very fancy, if you feel like implementing, you're very welcome to! But a few copy pastes is also fine :)